dynamodb-stream-reader/src/main/java/no/unit/nva/dynamodb/IndexFilterBuilder.java

```diff
@@ -0,0 +1,57 @@
+package no.unit.nva.dynamodb;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Predicate;
+
+public class IndexFilterBuilder {
+
+    private boolean allowAll;
+    private Set<String> wantedIndexes;
+
+    /**
+     * Adds index to set of wanted indexes.
+     * @param index to be added
+     * @return builder to use
+     */
+    public IndexFilterBuilder withIndex(String index) {
+        if (this.wantedIndexes == null) {
+            this.wantedIndexes = new HashSet<>();
+        }
+        this.wantedIndexes.add(index);
+        return this;  //By returning the builder each time, we can create a fluent interface.
```

Comment:
Remove comment for explaining Builder pattern

Reply:
BS comment removed (don't copy code without reading it)

Reply:
Comment still there. Builders are a common pattern. Comments should be avoided unless necessary